exercises/sort_by_pattern_test.rb

```diff
@@ -33,10 +33,11 @@ def test_sort_alphabetically_by_last_letter
   end
 
   def test_sort_by_distance
-    skip
     distances = ["1cm", "9cm", "30cm", "4cm", "2cm"]
     transformed = []
-    # Your code goes here
+    distances.each do |e|
+      transformed << [e.sub(/cm/, '').to_i, e]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.